### PR TITLE
[Tenant-Operator] Remove cert-manager

### PR DESCRIPTION
**Pull request by abdulhaseeb2**

### CODE REVIEW

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-254-3f604320`

File: content/sre/tenant-operator/installation.md [hunk 1]
```diff
@@ -123,5 +121,5 @@ A default `IntegrationConfig` is installed with tenant-operator, which can be fo
 
 ## Note
 
-* If tenant-operator is deployed in a newly created namespace. Then restart tenant-operators pod once so that tenant-operator can retrieve `webhook-server-cert` provided by certmanager(if pod started before secret was made).
+* If tenant-operator is deployed in a newly created namespace. Then restart tenant-operators pod once so that tenant-operator can retrieve `webhook-server-cert` provided by openshift(if pod started before secret was made).
```
Comment (ShahidHussain-srti) [inline content/sre/tenant-operator/installation.md]: grammar:
"If tenant-operator is deployed in a newly created namespace, restart its pod once so tenant-operator can retrieve `webhook-server-cert` provided by openshift(if the pod is started before the secret was made)."

**Review by ShahidHussain-srti — APPROVED** [review]

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-254-965c199f`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.189`

**Pull request merged**